hoot-core/src/main/cpp/hoot/core/io/OsmChangesetSqlFileWriter.cpp

```diff
@@ -170,13 +176,13 @@ ElementPtr OsmChangesetSqlFileWriter::_getChangeElement(ConstElementPtr element)
     default:
       throw HootException("Unknown element type");
   }
-
   return changeElement;
 }
 
 // If osm.changeset.file.writer.generate.new.ids is false, then these create methods assume
 // you've already set the ID correctly in terms of the OSM API target db for the element to be
 // created.
+//TODO: Can we get rid of the generate new ID's option now?
```

Comment:
I handled it in OsmChangesetXmlWriter.  The sql writer doesn't go through the rails port, so needs the positive ids.  But related to the question in the comment....I may need to keep that setting around for the sql writer and enable it to keep the id sequences for the new elements from having gaps...not sure... But since we're not using the sql writer in our workflow anyway, I won't worry about it for now.